camelot/admin/entity_admin.py

```diff
@@ -670,20 +670,19 @@ def decorate_search_query(self, query, text):
                         fa = related_admin.get_field_attributes(instrumented_attribute.key)
                         search_strategy = fa['search_strategy']
                         if search_strategy is not None:
-                            arg = search_strategy.get_clause(search_strategy, text, fa, instrumented_attribute)
+                            assert issubclass(search_strategy, list_filter.FieldSearch)
```

Comment:
looks good, voor mijn part leven we in de veronderstelling dat we die 'path' search er gewoon gaan uitsmijten

uiteindelijk moet je dan toch zo veel specialiseren om het performant te krijgen, dat je het beter direct kan doen